R/get.R

```diff
@@ -202,9 +195,13 @@ get_used_functions <- function(file){
       mutate(function_name = coalesce(.data[["SYMBOL_FUNCTION_CALL"]],
                                       .data[["SPECIAL"]]))
 
-   get_library(combine_tokens) %>%
-      select(all_of(c("function_name", "library"))) %>%
-      distinct()
+   distinct_use <- get_library(combine_tokens) %>%
+      select(.data$function_name, .data$library) %>%
+      distinct(across())
+
+   distinct_use[is.na(distinct_use)] <- "!!! NOT FOUND !!!"
+
+   distinct_use
```

Comment:
```suggestion
get_library(combine_tokens) %>%
      select(all_of(c("function_name", "library"))) %>%
      distinct()
```

Reply:
ummm....I would of thought our rewind would of fixed this.

Reply:
yeah, me too